app/components/billing/payment.js

```diff
@@ -85,7 +86,11 @@ export default Component.extend({
       } else {
         yield this.subscription.changePlan.perform(this.selectedPlan.id);
       }
-      yield this.accounts.fetchV2Subscriptions.perform();
+      if (!Ember.testing) {
+        yield this.pollSubscription.perform();
+      } else {
+        yield this.accounts.fetchV2Subscriptions.perform();
+      }
```

Comment:
This doesn't seem necessary to me, because `pollSubscription` task basically does the same. Although, this differentiation in tests makes them unreliable, because they may pass when actual behavior fails. Does it make sense?